### DISCUSSION TITLE
window resize with VUE 3

### DISCUSSION THREAD

**Discussion by murilolivorato** [quasarframework/quasar#16296] [Ideas / Proposals] [answered]
Does hava any how to do a action on windows resize , like this -
https://masteringjs.io/tutorials/vue/resize-event

with vue 3 ?

I did like this  -
```
onMounted(() => {
  window.addEventListener('resize', start)
})

onUnmounted(() => {
  window.removeEventListener('resize', start)
})


```

But it gave this error -

```
 TypeError: window.addEventListener is not a functio

```

**Comment by metalsadman** [comment]
https://vueuse.org/core/useResizeObserver/ https://vueuse.org/core/useWindowSize/

**Comment by murilolivorato** [comment, marked as answer]
thanks , I found quasar q-resize-observer as well .
